src/Netflix/Main.java

```diff
@@ -0,0 +1,237 @@
+package Netflix;
+
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Main Class File
+ * @author: Brighton Zhang
+ */
+
+public class Main {
+    static BufferedReader keyboard;
+    public static void main (String[] args) throws IOException {
+
+        // Variable and object instantiation
+        String strUser;
+        String strChoice;
+        int intFilm;
+        int intSlot;
+        int intCounter;
+        boolean blnSuperHero, blnCrime , blnAdventure, blnComedy, blnSciFi, blnDrama, blnHorror;
+        boolean blnMenu = true;
+        boolean blnList = false;
+        Scanner sc = new Scanner(System.in);        
+        ArrayList<String> filmList = new ArrayList<String>();
+        ArrayList<String> genreList = new ArrayList<String>();
+        ArrayList<Film> objFilmList = new ArrayList<Film>();
```

Comment:
the existence of `objFilmList`, which is more useful than your `filmList` variable, makes `filmList` redundant and unnecessary.  as mentioned in my previous comment, a UserList class could be much more ideal.